### DISCUSSION TITLE
Supporting attribute selectors

### DISCUSSION THREAD

**Discussion by liby** [johanholmerin/style9#70] [Ideas]
Currently, style9 does not seem to support attribute selectors. If I miss anything, please let me know.

```
export const styles = style9.create({
  a11y: {
     "a[data-a11y="true"]': {
      ...
    },
  },
});
```

**Comment by johanholmerin** [comment]
style9 does not support any selectors, only pseudo states and at-rules, as that would not be compatible with the deterministic style resolution.

**Reply by liby** [reply]
Okay, I get it.

So what are the best practices for trying to do something similar in DSS? 

For example, access data attribute with `ref` and then applying different class based on the values?

**Reply by johanholmerin** [reply]
You'll either need to query the attribute and apply the class based on that, or apply the class at the same time as rendering the attribute.
